flask_restplus/api.py

```diff
@@ -492,6 +492,7 @@ def handle_error(self, e):
             }
 
         default_data['message'] = default_data.get('message', str(e))
+        headers = default_data.pop("headers", headers)
```

Comment:
Looks great. I'll wait for `unpack` and rebase my branch from `master` and implement the changes.